lib/heuristics/scheduling_heuristic.rb

```diff
@@ -1194,5 +1218,17 @@ def restore
       @uninserted = @previous_uninserted
       @candidate_services_ids = @previous_candidate_service_ids
     end
+
+    def reject_all_visits(id, visits_number, specified_reason)
+      visits_number.times.each{ |visit_index|
+        @uninserted["#{id}_#{visit_index + 1}_#{visits_number}"] = {
+          original_service: id,
```

Comment:
original_service_id ?

Reply:
I was pointing `original_service` and not `id`

Reply:
This is named _original_service_ all over the code. I will make the clean in a new PR in order to check these modifications do not break anything.